src/Deployer.php

```diff
@@ -315,6 +315,8 @@ public static function run(string $version, ?string $deployFile)
                 $output->setVerbosity(OutputInterface::VERBOSITY_DEBUG);
             }
             self::printException($output, $exception);
+            
+            exit(1);
```

Comment:
Let’s use Symfony h onds fir this.

Reply:
Methods

Reply:
Do you mean Command constants? Cause your current console implementation can't rely on "returning" status code.

Reply:
Oh, I see.